flexible-classroom/develop/disable-whiteboard-module.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: "Disable the whiteboard module"
```

Comment:
Suggestion: Change the title to "Enable/Disable the whiteboard module".
Reason: Clarity.

Reply:
Changed to enable and disable